Externals/mbedtls/CMakeLists.txt

```
@@ -20,7 +20,7 @@
 #   mbedtls, mbedx509, mbedcrypto and apidoc targets.
 #
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5.0)
```

```suggestion
cmake_minimum_required(VERSION 2.8.12...4.1.1)
```
Would something like this make more sense. Support for 3.5.0 will inevitably be removed at some point as well causing these issues again sooner than later.
I think it makes more sense to specify that it works in the latest version?